src/main/kotlin/com/openlattice/shuttle/control/IntegrationUpdate.kt

```diff
@@ -0,0 +1,19 @@
+package com.openlattice.shuttle.control
```

Comment:
Needs a `StreamSerializer`

Reply:
Also shouldn't this be a `data class` ? 🤔

Reply:
Both accurate!